pilot/helpers/agents/test_Developer.py

```diff
@@ -105,23 +105,34 @@ def test_implement_task_reject_with_user_input(self, mock_completion, mock_save,
         # Then we include the user input in the conversation to update the task list
         assert mock_completion.call_count == 3
         prompt = mock_completion.call_args_list[2].args[0][2]['content']
-        assert prompt.startswith('''
-# Completed Task Steps:
-```
-[{'command': 'ls -al'}, {'command': 'ls -al src'}]
+        assert prompt.startswith('''Here is a list of completed steps:
 ```
 
-# Current Step:
-This step will not be executed. no, use a better command
+step #1:
+step_type: command_to_run: ``
+step #2:
+step_type: command_to_run: ``
+```
+Right now, we're working on the step #2:

```

Comment:
same as comment above, here it says list of completed steps: #1 and #2 and then it says right now, we're working on step #2.